packages/core/src/2d/text/TextStyle.ts

```diff
@@ -0,0 +1,63 @@
+import { assignmentClone } from "../../clone/CloneManager";
+import { UpdateFlag } from "../../UpdateFlag";
+import { UpdateFlagManager } from "../../UpdateFlagManager";
+
+export class TextStyle {
+  @assignmentClone
+  private _fontSize: number = 24;
+  @assignmentClone
+  private _isBold: boolean = false;
+  @assignmentClone
+  private _isItalic: boolean = false;
+
+  private _updateFlagManager: UpdateFlagManager = new UpdateFlagManager();
+
+  /**
+   * The font size of the TextRenderer.
+   */
+  get fontSize(): number {
+    return this._fontSize;
+  }
+
+  set fontSize(value: number) {
+    if (this._fontSize !== value) {
+      this._fontSize = value;
+      this._updateFlagManager.distribute();
+    }
+  }
```

Comment:
I think this in `TextRenderer` is better.

Reply:
done, now is _fontStyle, and type is enum FontStyle